pkg/certificate/types.go

```diff
@@ -122,6 +122,9 @@ type Manager struct {
 // MRCClient is an interface that can watch for changes to the MRC. It is typically backed by a k8s informer.
 type MRCClient interface {
 	List() ([]*v1alpha2.MeshRootCertificate, error)
```

Comment:
We could also change these methods to resemble the methods that are on the KubeClient, which would allow us to just use the compute interface directly

Reply:
Updated!